src/views/pages/Swap/Swap.js

```diff
@@ -722,7 +725,11 @@ const Swap = () => {
                             <div className="text-right">
                               ~$
                               {swapInput2?.value &&
-                                formatFromWei(getInput2USD())}
+                                formatFromWei(getInput2USD(), 2)}
+                              {' ('}
+                              {swapInput1?.value &&
+                                formatFromUnits(getRateSlip())}
+                              {'%)'}
```

Comment:
Add the above-explained usd-rate-slip into the UI and make the USD values rounded to 2 decimals)